clarity/src/vm/functions/mod.rs

```diff
@@ -517,6 +523,36 @@ pub fn lookup_reserved_functions(name: &str, version: &ClarityVersion) -> Option
                 SpecialFunction("from_consensus_buff", &conversions::from_consensus_buff)
             }
             ReplaceAt => SpecialFunction("replace_at", &sequences::special_replace_at),
+            BitwiseAND => NativeFunction(
+                "native_bitwise_and",
+                NativeHandle::DoubleArg(&arithmetic::native_bitwise_and),
+                ClarityCostFunction::BitwiseAND,
+            ),
+            BitwiseOR => NativeFunction(
+                "native_bitwise_or",
+                NativeHandle::DoubleArg(&arithmetic::native_bitwise_or),
+                ClarityCostFunction::BitwiseOR,
+            ),
+            BitwiseNOT => NativeFunction(
+                "native_bitwise_or",
```

Comment:
Typo (logic error): this should be "native_bitwise_not"

Reply:
Good catch! Fixed.